include/triggers.h

```diff
@@ -18,6 +18,29 @@
 // setai
 // route
 
+typedef enum c3_condition_midi_t
+{
+  C3_COND_DIRE = 0,
+  C3_COND_FLIGHT,
+  C3_COND_WIN,
+  C3_COND_LOSE,
+  C3_COND_COMBAT,
+  C3_COND_PAD,
+  C3_COND_SCARY,
+}
+c3_condition_midi_t;
+
+static const char *c3_condition_midi_text[] =
+{
+  "dire.mid\0",
```

Comment:
i believe c adds the null byte for you if you don't explicitly allocate the array of chars short of one byte for the last character. the addition of the explicit "\0" is not needed.